### DISCUSSION TITLE
Disable YouTube cache trimming during playback

### DISCUSSION THREAD

**Discussion by Azirine** [mpv-player/mpv#15359] [Q&A] [answered]
Which option disables YouTube cache trimming during playback? For example, here I played the video from the beginning but the front portion of the cache is already gone.

![screenshot](https://github.com/user-attachments/assets/85fe7270-73d9-4aa7-9bbd-367a00f73e2b)

**Comment by llyyr** [comment, marked as answer]
https://mpv.io/manual/master/#options-demuxer-max-back-bytes
